piperider_cli/generate_report.py

```diff
@@ -122,3 +122,8 @@ def output_report(target_directory):
             console.print(f"Report generated in {output}/index.html")
         else:
             console.print(f"Report generated in {default_output_directory}/index.html")
+
+        # only open the local file report if auto-upload is OFF
+        if open_report and not auto_upload:
```

Comment:
browsers would open a local report and a cloud report when giving`--open` and `--upload` at the same time

Reply:
Maybe I'm misreading this case, but it works on my side though 🤔 . 
` piperider run --open --upload  # this only opens in cloud on my local`

Based on my understanding, this statement should ONLY allow local report to open when either `--open` or `upload` is  `None`